CHANGELOG

```diff
@@ -1,3 +1,8 @@
+Release 0.35.0
+
+- Added to metrics info about running MongoDB version and Mongo vendor
```

Comment:
Can be reworded to _**Added version and vendor information to metrics info**_

Reply:
Done